scripts/tools/silabs/FactoryDataProvider.py

```diff
@@ -0,0 +1,212 @@
+import subprocess
+import sys
+import argparse
+import struct
+
+# global variables
+Discriminator_ui = 0
+Passcode_ui = 0
+IterationCount_ui = 0
+Salt_str = ""
+Verifier_str = ""
```

Comment:
Global module variables should probably have `_` prefix and no capital letter at the start